package.json

```diff
@@ -8,108 +8,110 @@
-    "bloodhound-js": "^1.2.1",
-    "bootstrap": "^3.3.7",
+    "babel-loader": "8.0.6",
+    "babel-plugin-transform-decorators-legacy": "1.3.5",
+    "babel-plugin-transform-object-assign": "6.8.0",
+    "babel-plugin-transform-runtime": "6.15.0",
+    "babel-polyfill": "6.26.0",
+    "backbone": "1.3.3",
+    "backbone-associations": "0.6.2",
+    "backbone.localstorage": "1.1.16",
+    "bloodhound-js": "1.2.3",
+    "bootstrap": "3.3.7",
     "classnames": "latest",
-    "codemirror": "^5.42.0",
-    "copy-webpack-plugin": "^4.6.0",
-    "create-react-class": "^15.6.3",
-    "css-loader": "0.28.7",
-    "d3": "^4.2.7",
-    "d3-plugins-dist": "^3.2.0",
-    "dat-gui": "^0.5.0",
-    "date-fns": "^2.0.0-alpha.16",
-    "detector-webgl": "^2.0.0",
-    "eslint": "5.13.0",
-    "exports-loader": "^0.6.3",
-    "express": "^4.16.4",
-    "express-ws": "^2.0.0",
-    "extract-text-webpack-plugin": "3.0.0",
+    "codemirror": "5.42.0",
+    "copy-webpack-plugin": "4.6.0",
+    "create-react-class": "15.6.3",
+    "css-loader": "3.0.0",
+    "d3": "4.2.7",
+    "d3-plugins-dist": "3.2.0",
+    "dat.gui": "0.7.6",
+    "date-fns": "2.0.0-alpha.16",
+    "detector-webgl": "2.0.0",
+    "eslint": "6.0.1",
+    "exports-loader": "0.6.3",
+    "express": "4.16.4",
+    "express-ws": "2.0.0",
     "file-loader": "2.0.0",
-    "file-saver": "^1.3.3",
-    "google-maps": "^3.2.1",
+    "file-saver": "1.3.3",
+    "google-maps": "3.2.1",
     "griddle-0.6-fork": "https://github.com/rodriguez-facundo/griddle.git",
-    "handlebars": "^4.0.5",
+    "handlebars": "4.0.5",
     "html2canvas": "1.0.0-alpha.12",
-    "html-webpack-plugin": "2.30.1",
-    "ignore-loader": "^0.1.2",
-    "imports-loader": "^0.7.0",
-    "jasmine": "^3.2.0",
-    "jquery": "^1.9.1",
+    "html-webpack-plugin": "3.2.0",
+    "ignore-loader": "0.1.2",
+    "imports-loader": "0.7.0",
+    "jasmine": "3.2.0",
+    "jquery": "3.4.0",
     "jquery-ui-bundle": "1.12.1",
-    "js-cookie": "^2.1.3",
-    "json-loader": "^0.5.4",
-    "jszip": "^3.1.3",
-    "less": "^2.7.2",
-    "less-loader": "4.0.5",
-    "lg-fullscreen": "^1.0.1",
-    "lg-pager": "^1.0.2",
-    "lg-thumbnail": "^1.1.0",
-    "lg-video": "^1.1.0",
-    "lightgallery": "^1.6.0",
-    "material-ui-pickers": "^1.0.0-rc.17",
-    "mathjs": "^3.5.3",
-    "openseadragon": "^2.2.1",
-    "pako": "^1.0.3",
+    "js-cookie": "2.1.3",
+    "json-loader": "0.5.4",
+    "jszip": "3.2.1",
+    "less": "2.7.2",
+    "less-loader": "5.0.0",
+    "lg-fullscreen": "1.0.1",
+    "lg-pager": "1.0.2",
+    "lg-thumbnail": "1.1.0",
+    "lg-video": "1.1.0",
+    "lightgallery": "1.6.0",
+    "mathjs": "3.5.3",
+    "mini-css-extract-plugin" : "0.7.0",
+    "openseadragon": "2.2.1",
+    "pako": "1.0.3",
     "pixi.js": "4.2.3",
-    "plotly.js": "^1.42.5",
+    "plotly.js": "1.42.5",
     "prop-types": "latest",
-    "qunitjs": "^2.1.1",
-    "raw-loader": "^0.5.1",
-    "react": "^16.6.3",
-    "react-dnd": "7.0.2",
+    "qunit": "2.9.2",
```

Comment:
@jrmartin are we still using qunit?

Reply:
@tarelli There's only one test left from qunit days, but the tests happens in casper too. So we can remove it now.

Reply:
Yeah I'd say let's remove it and the dependency then.